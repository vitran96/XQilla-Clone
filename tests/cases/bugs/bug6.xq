declare default element namespace "http://default.org/ns";
declare namespace a = "http://custom.org/ns";

<elem>test</elem>, 
element elem {'test'},
element a:elem {'test'},
<a:node> {attribute attr {'test'}}</a:node>,
<a:node attr="test"></a:node>,
<a:node> {attribute { fn:QName("http://anamespa.ce", "attr") } {'test'}}</a:node>,
<a:node> {attribute { fn:QName("http://custom.org/ns", "attr") } {'test'}}</a:node>,
<newElement xmlns="www.mynamespace.com">{element original {"Original Content"}}</newElement>/*
