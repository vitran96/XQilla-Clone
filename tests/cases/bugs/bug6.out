<elem xmlns="http://default.org/ns">test</elem><elem xmlns="http://default.org/ns">test</elem><a:elem xmlns:a="http://custom.org/ns">test</a:elem><a:node xmlns:a="http://custom.org/ns" attr="test"/><a:node xmlns:a="http://custom.org/ns" attr="test"/><a:node xmlns:a="http://custom.org/ns" xmlns:ns_1="http://anamespa.ce" ns_1:attr="test"/><a:node xmlns:a="http://custom.org/ns" a:attr="test"/><original xmlns="www.mynamespace.com">Original Content</original>